app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,10 @@
+Name: <%= @recipe.name %>
+Complexity: <%= @recipe.complexity %>
+Genre: <%= @recipe.genre %>
+
+Ingredients:
+<% @recipe.ingredients.each do |ingredient| %>
+  <%= ingredient.name %>
+<% end %>
+
+Total Cost: <%= @recipe.total_cost %>
```

Comment:
Awesome job creating a clean view that calls on model methods to do the heavy lifting with data!